frontend/src/actions/homeActions.js

```diff
@@ -0,0 +1,94 @@
+import * as API_ROUTES from "@/utils/apiConstants";
+import * as TYPES from "@/actions/types.js";
+
+import API from "@/utils/axiosInstance";
+import { DEFAULT_PER_PAGE } from "@/assets/constants/paginationConstants";
+import { showFailureToast } from "@/actions/toastActions";
+
+export const fetchOCPJobsData = () => async (dispatch, getState) => {
```

Comment:
This is the only stub making calls to the backend. But I am not sure if it is able to plugin OCP and Quay data as well to the frontend. If not here can you please help me understand where is it happening in the frontend code?

Reply:
This PR is for Home page so for OCP and Quay data new PRs will be raised.